shared/src/persistence/elasticsearch/getReadyForTrialCases.js

```diff
@@ -17,13 +17,65 @@ exports.getReadyForTrialCases = async ({ applicationContext }) => {
       body: {
         _source: ['docketNumber'],
         query: {
-          term: {
-            'status.S': CASE_STATUS_TYPES.generalDocket,
+          bool: {
+            filter: [
+              {
+                term: {
+                  'entityName.S': 'DocketEntry',
+                },
+              },
+            ],
+            must: [
+              {
+                term: {
+                  'documentType.S': 'Answer',
+                },
+              },
+              {
+                range: {
+                  'createdAt.S': {
+                    // Getting one extra day just so we don't miss any.
+                    // Case.prototype.checkForReadyForTrial will continue to do the math as it always has.
+                    lte: 'now-44d/d',
+                  },
+                },
+              },
+              {
+                has_parent: {
+                  inner_hits: {
+                    _source: {
+                      includes: ['caseCaption', 'docketNumber', 'status'],
+                    },
+                    name: 'case-mappings',
+                  },
+                  parent_type: 'case',
+                  query: {
+                    bool: {
+                      must: [
+                        {
+                          term: {
+                            'status.S': CASE_STATUS_TYPES.generalDocket,
+                          },
+                        },
+                      ],
+                    },
+                  },
+                  score: true,
+                },
+              },
+            ],
+            must_not: [
+              {
+                term: {
+                  'isStricken.BOOL': true,
```

Comment:
do we care if an answer is isStricken?

Reply:
Confirming that Chris Holly says the stricken docket entries are to be treated as if they never happened, so we do want to exclude them.